Assets/FluidBehaviorTree/Runtime/BehaviorTree/Builder/BehaviorTreeBuilder.cs

```diff
@@ -62,6 +62,18 @@ public class BehaviorTreeBuilder {
             return ParentTask<ReturnFailure>(name);
         }
         
+        public BehaviorTreeBuilder RepeatUntilSuccess (string name = "repeat until success") {
```

Comment:
Also adding an icon is suggested for the new tree visualizer. Currently using Material Icons. One like this should work for all 3 https://material.io/tools/icons/?search=repeat&icon=repeat&style=baseline

Reply:
PS You'll need the latest `develop` for icons.